Podar Delia, Turbuleasa Daria - m42

PodarDelia opened the pull request.

PodarDelia committed:
Podar Delia, Turbuleasa Daria - m42

APGroza reviewed (changes requested):
implies  Mosquito (all transmits Covid-19)) inseamna ca Tantarii transmit doar COVID-19. Ei transmi si altceva. Mitul spune ceva de genul 

implies  Mosquito (some transmits Covid-19)) sau 
implies  Mosquito (at-most transmits Covid-19))

PodarDelia committed:
Update m42.racer

Changed line (implies  Mosquito (all transmits Covid-19)) with (implies  Mosquito (some transmits Covid-19))

APGroza committed:
Merge branch 'master' into master
